controlcenter/static/controlcenter/css/all.css

```diff
@@ -308,15 +315,15 @@
   }
 
   .controlcenter__masonry__block--w2 {
-    width: 33.333333333333336%;
+    width: 33.33333333333333%;
   }
 
   .controlcenter__masonry__block--w3 {
     width: 50%;
   }
 
   .controlcenter__masonry__block--w4 {
-    width: 66.66666666666667%;
+    width: 66.66666666666666%;
   }
```

Comment:
Which browser di you use? Have you checked other browsers then? This can brake things.